trestle/utils/fs.py

```diff
@@ -93,30 +93,18 @@ def has_parent_path(sub_path: pathlib.Path, parent_path: pathlib.Path) -> bool:
     return True
 
 
-def get_contextual_model_type(path: pathlib.Path = None) -> Tuple[Type[OscalBaseModel], str]:
-    """Get the full contextual model class and full jsonpath for the alias based on the contextual path."""
-    logger.debug(f'get contextual model type for input path {path}')
-
-    if path is None:
-        path = pathlib.Path.cwd()
-    else:
-        if not path.exists():
-            path = pathlib.Path.cwd() / path
-
-    logger.debug(f'get contextual model type final path {path}')
-
-    if not is_valid_project_model_path(path):
-        raise err.TrestleError(f'Trestle project model not found at {path}')
-
-    root_path = get_trestle_project_root(path)
-    project_model_path = get_project_model_path(path)
+def get_relative_model_type(relative_path: pathlib.Path) -> Tuple[Type[OscalBaseModel], str]:
+    """
+    Given the relative path of a file with respect to 'trestle_root' return the oscal model type.
 
-    logger.debug(f'root_path is {root_path} and project_model_path is {project_model_path}')
+    Args:
+        relative_path: Relative path of the model of interest with respect to the root directory of the trestle project.
 
-    if root_path is None or project_model_path is None:
-        raise err.TrestleError('Trestle project model not found')
+    Returns:
+        Type of Oscal Model for the provided model
+        Alias of that oscal model.
 
-    relative_path = path.relative_to(str(root_path))
+    """
     project_type = relative_path.parts[0]  # catalogs, profiles, etc
```

Comment:
I'm not sure how unhappy cases are handled.  What if the first part of path is foo?  Also what if you have catalogs/catalog.json?  i.e. valid object in wrong part of directory.

Reply:
Now the project will choke if the incorrect directory is passed. This relative directory always relative to trestle project root directory.